### PR TITLE
Update getting started pages

### PR DESCRIPTION
**Pull request by aeslaughter**
Updates to the MooseDocs getting started pages.
(refs #9638)

### CODE REVIEW

**Comment by aeslaughter** [comment]
Whoever wants to take a look, the new "Getting Started" instructions can be found here: http://mooseframework.org/docs/PRs/10070/site/getting_started/installation/mac_os/index.html

**Review by brianmoose — CHANGES_REQUESTED** [review]
None of the package links seem to work

**Review by brianmoose — DISMISSED** [review]

**Comment by brianmoose** [comment]
The package links seem to work now, @milljm should probably review the content

File: docs/content/getting_started/installation/mac_pre_req.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+## Pre-Requisites
+
+##### (1) Xcode Command Line Tools
+
+To install Command Line Tools (CLT) on your machine, simply open a terminal and run the following command.
+
+```bash
+xcode-select
```
Comment (milljm) [inline docs/content/getting_started/installation/mac_pre_req.md]: I believes its:
```bash
xcode-select --install
```

**Review by milljm — DISMISSED** [review]
Everything else looks fine!

**Review by permcody — DISMISSED** [review]
Looks OK except you should change Pre-Requisite back to prerequisite throughout.

**Review by permcody — APPROVED** [review]

**Pull request merged**